frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -366,11 +367,18 @@ func (h *startCVDHandler) Launch(instanceNumber uint32, artifactsDir, homeDir st
 	return nil
 }
 
-func createDir(dir string, failIfExist bool) error {
-	// TODO(b/238431258) Use `errors.Is(err, fs.ErrExist)` instead of `os.IsExist(err)`
-	// once b/236976427 is addressed.
+func createDirFailIfExist(dir string) error {
+	err := os.Mkdir(dir, 0774)
+	if err != nil {
+		return err
+	}
+	// Mkdir set the permission bits (before umask)
+	return os.Chmod(dir, 0774)
```

Comment:
nit: "regardless of `umask`" is more accurate than "before `umask`" since `chmod` doesn't take `umask` into account at all.

Reply:
I see, the comment was referring to the `os.Mkdir` call above which aims to set the same permission bits as well, clarifying the need for the `os.Chmod` call. I rephrased the comment a little bit. WDYT?

Reply:
I didn't realized the comment said "Mkdir sets...". This comment could just be "Set dir permission regardless of umask".

Reply:
done!